sdk/batch/Microsoft.Azure.Batch.FileStaging/Directory.Build.targets

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.targets))\Directory.Build.targets" />
+
+  <Target Name="ValidateTargetFrameworks" BeforeTargets="Build">
```

Comment:
You should not be overriding this target without a valid reason.

Reply:
As mentioned in the previous threads, in order to get rid of support of .netstandard1.4 and .net452 without deriving the target frameworks from eng/Directory.Build.Common.targets, we had to locally define this target file.